File: challenges/2-tabuada/python/Apenas-Kev/Tabuada.py
while True:
    try:
        numero = int(input('Insira o numero para receber a tabuada: '))
        break;
    except ValueError:
        print('Por favor, insira apenas numeros inteiros')
ctrl = 0
print('='*27)
while (numero > 0):
    ctrl = (ctrl + 1)
    print(f'A tabuada do numero {ctrl} é:\n  {ctrl}*1 = {ctrl}\n  {ctrl}*2 = {ctrl*2}\n  {ctrl}*3 = {ctrl*3}\n  {ctrl}*4 = {ctrl*4}\n  {ctrl}*5 = {ctrl*5}\n  {ctrl}*6 = {ctrl*6}\n  {ctrl}*7 = {ctrl*7}\n  {ctrl}*8 = {ctrl*8}\n  {ctrl}*9 = {ctrl*9}\n  {ctrl}*10 = {ctrl*10}\n{"="*27}')
    numero = numero - 1

File: challenges/2-tabuada/js/afpp3/tabuada.js
const numero = +process.argv.slice(2);

function tabuada(numero) {
  if (numero !== isNaN && Number.isInteger(numero)) {
    for (let i = 1; i <= numero; i++) {
      console.log(`\n===========================`);
      console.log(`A tabuada do numero ${i} é:\n`);
      for (let count = 1; count <= 10; count++) {
        console.log(`${i} * ${count} = ${i * count}`);
      }
    }
  } else {
    console.error(
      "❌ Por favor, digite apenas números inteiros como argumento"
    );
  }
}

tabuada(numero);
